src/Propel/Runtime/Map/DatabaseMap.php

```diff
@@ -120,6 +134,41 @@ public function addTableFromMapClass(string $tableMapClass): TableMap
         return $this->getTable($table->getName());
     }
 
+    /**
+     * Registers a table map classes (by qualified name) as table belonging
+     * to this database.
+     *
+     * Classes added like this will only be instantiated when accessed
+     * through {@link DatabaseMap::getTable()},
+     * {@link DatabaseMap::getTableByPhpName()}, or
+     * {@link DatabaseMap::getTables()}
+     *
+     * @param class-string $tableMapClass The name of the table map to add
+     *
+     * @return void
+     */
+    public function registerTableMapClass($tableMapClass): void
```

Comment:
should this have a type?

Reply:
I had it as string, but then cs-fix added that into the @param declaration, so I removed it again. Now I have added it again, let me know if you do prefer it the other way,